integration_test/upgrade_test.go

```diff
@@ -45,15 +45,23 @@ func TestKrewUpgrade(t *testing.T) {
 
 	// plugins installed via manifest get the special "detached" index so this needs to
 	// be changed to default in order for it to be upgraded here
-	receipt := environment.NewPaths(test.Root()).PluginInstallReceiptPath(validPlugin)
-	modifyReceiptIndex(t, receipt, "default")
+	receiptPath := environment.NewPaths(test.Root()).PluginInstallReceiptPath(validPlugin)
+	modifyReceiptIndex(t, receiptPath, "default")
+	pluginReceipt := test.loadReceipt(receiptPath)
+	initialCreationTimestamp := pluginReceipt.CreationTimestamp
```

Comment:
We don't need `pluginReceipt`. So let's simplify to
```suggestion
	initialTimestamp := test.loadReceipt(receiptPath).CreationTimestamp
```
(same below)